Form-Controls/styles.css

```diff
@@ -0,0 +1,133 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto";
+}
+.fxdc-jcc-aic {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
The page cannot extend to screen width and height, is there any way to improve it?
Can we make the page more responsive?

Reply:
This code is for making similar functions, not for extending the container.  I should have assigned the height to each container.

Reply:
I see, I should have referred to the root element. Why the root element cannot take the full screen width and height?

Reply:
I am not sure if I misunderstood? root element should be html...?

Reply:
Yes, `<html lang="en" class="fxdc-jcc-aic"> `

Reply:
You mean I should not put height and width on the "html" element?

Reply:
please refer to https://github.com/CodeYourFuture/HTML-CSS-Challenges/pull/181/commits/b5447c9dd79bbccbddd9a84b7031d42e3ceb5f6f#r1024587990

Reply:
Sorry, now i understood your meaning...actually why we have to "full screen" for extending the container ? I feel it is not good look..